types/mvdan-sh/mvdan-sh-tests.ts

```diff
@@ -55,3 +55,10 @@ parser.Interactive(src, (stmts: sh.Stmt[]) => {
     const incomplete = parser.Incomplete(); // $ExpectType boolean
     return true;
 });
+
+// Test syntax.IsIncomplete
+try {
+	parser.Parse("echo ${", "src");
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/mvdan-sh/mvdan-sh-tests.ts;
  Line: 61;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.